packages/xo-web/src/common/modal.js

```diff
@@ -381,12 +381,15 @@ export const FormModal = decorate([
         <form id={state.formId}>
           {/* It should be better to use a computed to avoid calling the render function on each render,
             but reaclette(v0.4.0) not allow us to access to the effects from a computed */}
-          {state.component ||
-            (state.render !== undefined &&
-              state.render({
-                onChange: effects.onChange,
-                value: state.value,
-              }))}
+          {state.component !== undefined ? (
+            <state.component onChange={effects.onChange} value={state.value} />
+          ) : (
+            state.render !== undefined &&
+            state.render({
+              onChange: effects.onChange,
+              value: state.value,
+            })
+          )}
```

Comment:
We are changing the behavior of this function, `component` should be an instance, not a component class.

We can make this change if we have good reasons to, but we need to make sure it's not used elsewhere, and maybe make it more obvious by renaming the param to `Component`.